drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -517,16 +619,18 @@ static const struct rtc_stm32_config rtc_config = {
 	.pclken = rtc_clk,
 };
 
-
 static const struct counter_driver_api rtc_stm32_driver_api = {
-		.start = rtc_stm32_start,
-		.stop = rtc_stm32_stop,
-		.get_value = rtc_stm32_get_value,
-		.set_alarm = rtc_stm32_set_alarm,
-		.cancel_alarm = rtc_stm32_cancel_alarm,
-		.set_top_value = rtc_stm32_set_top_value,
-		.get_pending_int = rtc_stm32_get_pending_int,
-		.get_top_value = rtc_stm32_get_top_value,
+	.start = rtc_stm32_start,
+	.stop = rtc_stm32_stop,
+	.get_value = rtc_stm32_get_value,
+#ifdef CONFIG_COUNTER_RTC_STM32_SUBSECONDS
+	.get_value_64 = rtc_stm32_get_value_64,
+#endif /* CONFIG_COUNTER_RTC_STM32_SUBSECONDS */
+	.set_alarm = rtc_stm32_set_alarm,
+	.cancel_alarm = rtc_stm32_cancel_alarm,
+	.set_top_value = rtc_stm32_set_top_value,
+	.get_pending_int = rtc_stm32_get_pending_int,
+	.get_top_value = rtc_stm32_get_top_value,
```

Comment:
Non blocking: Formatting changes should be made in their own commit.

Reply:
Done.